### ISSUE TITLE
Train using darknet command line

### ISSUE THREAD

**Issue by mon28** [RobotEdh/Yolov-4#2]
Hi,

Do you know how i can train using AlexAB's compiled darknet on some other system?
Say I cloned and complied the darknet on system A. Now I want to ship this darknet on another system so that I can run train commands there with appropriate initial weights file, cfg file and data as per the format. Which all files do I need to copy for the same? Do i have to set any environment variables? If yes, what are they and which supporting libraries would i need?

Also waiting for a tensorflow wrapper that works just like the original darknet for training. So far everything that there is is shit! 

Thanks for your efforts!

**Comment by RobotEdh** [comment]
Hello

I have just released a procedure to train Yolov4 with Tensorflow based on Pascal VOC dataset

But you can adapt it for other datasets the main difficulty for me was to create a custom loss function for Yolov4

enjoy !
